README.md

```diff
@@ -87,96 +82,46 @@ Use dash (`-i -`) to read from standard input:
-* mixed - replay the recorded data based on exact matching when it is possible
 
-The OpenAPI schemas can be manually edited to mix the two modes.
+For more information about building, including mixing together the two modes and editing the created OpenAPI schema, see the [building documentation](./guides/BUILD.md).
 
-### Reading from Kafka
+## Mock
 
-Set `--source kafka` in `build` command:
-
-```bash
-$ meeshkan build --source kafka [-o path/to/output_directory]
-```
-
-_TODO: Configuration for Kafka_
-
-## Mocking
-
-You can use an existing OpenAPI spec, an OpenAPI spec you've built using `meeshkan build`, and recordings to create a mock server using Meeshkan.
+You can use an OpenAPI spec, such as the one created with `meeshkan build` to create a **mock** server using Meeshkan.
 
 ```bash
 $ pip install meeshkan # if not installed yet
 $ meeshkan mock
 ```
 
-### Common command-line arguments
-
-The following commands are available in mock mode:
-
-| Argument     | Description | Default |
-| ------------ | ----------- | ------- |
-| `port`       | Server port | 8000    |
-| `admin_port` | Admin port  | 8999    |
-| `log_dir`    | The directory containing `.jsonl` files for mocking directly from recorded fixtures | `logs` |
-| `specs_dir`  | The directory containing `.yml` or `.yaml` OpenAPI specs used for mocking, including ones built using `meeshkan build` | `specs` |
-
-## Recording
-
-In addition to the builder, Meeshkan provides a recorder that can capture API traffic using a proxy and, like the builder, automatically assembles it into an OpenAPI schema in addition to storing the raw recordings.
-
-```bash
-$ pip install meeshkan # if not installed yet
-$ meeshkan record
-```
-
-This starts Meeshkan as a reverse proxy on the default port of `8000`.  For example, with curl, you can use Meeshkan as a proxy like so.
-
-```bash
-$ curl http://localhost:8000/http://api.example.com
-```
-By default the recording proxy uses the first path items to navigate to a target host.
-Now you should have the `logs` folder with jsonl files and the `__unmock__` folder with ready openapi schemes. 
-
-For more advanced information about recording, including custom middleware, see the [server documentation](./meeshkan/server/SERVER.md).
+For more information about mocking, including adding custom middleware and modifying the mocking schema JIT via an admin API, see the [mocking documentation](./guides/MOCK.md).
 
-## Converting
-
-Meeshkan provides utilities to convert from certain popular recording formats to the `.jsonl` format.
-
-### Converting from pcap
+## Development
 
-You can convert [packet capture files](https://en.wikipedia.org/wiki/Pcap) to [HTTP Types](https://meeshkan.github.io/http-types/) format using the `convert` command:
+To obtain a copy of this source code, you can clone it from this git repository.
 
 ```bash
-meeshkan convert -i /path/to/file.pcap -o recordings.jsonl
+$ git clone http://github.com/meeshkan/meeshkan
```

Comment:
Do we need to give instructions for how to clone a repo? Especially as no-one probably wants to clone across plain http usually.